server/blob/blob-s3/src/main/java/org/apache/james/blob/objectstorage/aws/S3BlobStoreDAO.java

```diff
@@ -420,7 +418,7 @@ public Publisher<BucketName> listBuckets() {
         return Mono.fromFuture(client::listBuckets)
             .flatMapIterable(ListBucketsResponse::buckets)
             .map(Bucket::name)
-            .handle((bucket, sink) -> bucketNameResolver.unresolve(BucketName.of(bucket))
+            .handle((bucket, sink) -> bucketNameResolver.unresolve(ResolvedBucketName.of(bucket))
```

Comment:
**❔ question:** ‏same as above, I realize this was the case before this MR but is it normal that we don't go through 
`ResolvedBucketName resolvedBucketName = bucketNameResolver.resolve(bucketName);` here whil we do for all other public calls ? I don't necessarily expect this to be fixed in this MR, just trying to understand if there is something that needs fixing at all :D

Reply:
listBuckets returns some BucketName but list from S3 returns resolved bucket names.